src/tests/ftest/pool/RebuildTests.py

```diff
@@ -122,63 +120,42 @@ def test_simple_rebuild(self):
                 self.fail("Rebuilt objs not zero.\n")
             if pool.pool_info.pi_rebuild_st.rs_rec_nr != 0:
                 self.fail("Rebuilt recs not zero.\n")
-            pool_version = pool.pool_info.pi_rebuild_st.rs_version
 
             # create a container
-            container = DaosContainer(self.Context)
+            container = DaosContainer(self.context)
             container.create(pool.handle)
 
             # now open it
             container.open()
 
-            # how many objects and records are we creating
-            objcount = self.params.get("objcount",
-                                       '/run/testparams/numobjects/*')
-            reccount = self.params.get("reccount",
-                                       '/run/testparams/numrecords/*')
-            if objcount == 0:
-                reccount = 0
-
-            # which rank to write to and kill
-            rank = self.params.get("rank", '/run/testparams/ranks/*')
-
-            # how much data to write with each key
-            size = self.params.get("size", '/run/testparams/datasize/')
-
             saved_data = []
-            for i in range(0, objcount):
+            for _objc in range(self.objcount):
                 obj = None
-                for j in range(0, reccount):
-
+                for _recc in range(self.reccount):
                     # make some stuff up and write
-                    dkey = ''.join(random.choice(string.ascii_uppercase +
-                                                 string.digits)
+                    dkey = ''.join(random.choice(string.ascii_uppercase + string.digits)
                                    for _ in range(5))
-                    akey = ''.join(random.choice(string.ascii_uppercase +
-                                                 string.digits)
+                    akey = ''.join(random.choice(string.ascii_uppercase + string.digits)
                                    for _ in range(5))
-                    data = ''.join(random.choice(string.ascii_uppercase +
-                                                 string.digits)
-                                   for _ in range(size))
+                    data = ''.join(random.choice(string.ascii_uppercase + string.digits)
+                                   for _ in range(self.size))
 
-                    obj, tx = container.write_an_obj(data, len(data),
-                                                        dkey, akey, obj, rank)
+                    obj, tx = container.write_an_obj(data, len(data), dkey, akey, obj, self.rank,
```

Comment:
(pylint-line-too-long) Line too long (97/80)

(pylint-invalid-name) Invalid variable name "tx"

Reply:
no belong to current patch.